sql/2024/fonts/development/fonts_variable_by_service.sql

```diff
@@ -0,0 +1,43 @@
+-- Section: Development
+-- Question: Who is serving variable fonts?
+
+CREATE TEMPORARY FUNCTION SERVICE(url STRING) AS (
+  CASE
+    WHEN REGEXP_CONTAINS(url, r'((use|fonts)\.typekit\.(net|com))|webfonts\.creativecloud\.com') THEN 'Adobe'
```

Comment:
@bramstein, is the last one attributed correctly? It was under `edgefonts.net` in 2022.

Reply:
And in general, does this list need any update since 2022?

Reply:
Edge Fonts is a separate (free) web font service by Adobe, but it has been deprecated. As far as I know it still serves fonts however.

I'm not aware of any new web font service used at a scale significant enough that it would show up here.

Reply:
I see. Where should I put `webfonts.creativecloud.com`? Is it part of Edge Fonts?

Reply:
Or we can group Edge Fonts with Adobe if appropriate.